cfme/tests/cloud/test_providers.py

```diff
@@ -259,3 +259,13 @@ def test_api_port_max_character_validation(request):
 
     request.addfinalizer(prov.delete_if_exists)
     prov.create()
+
+
+@pytest.mark.ignore_stream("5.2", "5.3", "5.4")
```

Comment:
I have seen this cause a problem before, wouldn't an uncollectif  > 5.5 be better here?